`pandas>=1.1.0` and `python-dateutil>=2.7.3` are required for the following unit test to pass
```
_______________________________________________________________________________________ TestRowIterator.test_to_dataframe_tqdm_error ________________________________________________________________________________________

self = <tests.unit.test_table.TestRowIterator testMethod=test_to_dataframe_tqdm_error>

    @unittest.skipIf(pandas is None, "Requires `pandas`")
    @unittest.skipIf(tqdm is None, "Requires `tqdm`")
    @mock.patch("tqdm.tqdm_gui", new=None)  # will raise TypeError on call
    @mock.patch("tqdm.tqdm_notebook", new=None)  # will raise TypeError on call
    @mock.patch("tqdm.tqdm", new=None)  # will raise TypeError on call
    def test_to_dataframe_tqdm_error(self):
        from google.cloud.bigquery.schema import SchemaField
    
        schema = [
            SchemaField("name", "STRING", mode="REQUIRED"),
            SchemaField("age", "INTEGER", mode="REQUIRED"),
        ]
        rows = [
            {"f": [{"v": "Phred Phlyntstone"}, {"v": "32"}]},
            {"f": [{"v": "Bharney Rhubble"}, {"v": "33"}]},
            {"f": [{"v": "Wylma Phlyntstone"}, {"v": "29"}]},
            {"f": [{"v": "Bhettye Rhubble"}, {"v": "27"}]},
        ]
        path = "/foo"
    
        for progress_bar_type in ("tqdm", "tqdm_notebook", "tqdm_gui"):
            api_request = mock.Mock(return_value={"rows": rows})
            row_iterator = self._make_one(_mock_client(), api_request, path, schema)
    
            with warnings.catch_warnings(record=True) as warned:
                df = row_iterator.to_dataframe(
                    progress_bar_type=progress_bar_type,
                    create_bqstorage_client=False,
                )
    
            self.assertEqual(len(df), 4)  # all should be well
    
            # Warn that a progress bar was requested, but creating the tqdm
            # progress bar failed.
            for warning in warned:
>               self.assertIs(warning.category, UserWarning)
E               AssertionError: <class 'DeprecationWarning'> is not <class 'UserWarning'>

tests/unit/test_table.py:3284: AssertionError
```

--- a/testing/constraints-3.7.txt
+++ b/testing/constraints-3.7.txt
@@ -16,12 +16,12 @@ ipython==7.0.1
 opentelemetry-api==1.1.0
 opentelemetry-instrumentation==0.20b0
 opentelemetry-sdk==1.1.0
-pandas==1.0.0
+pandas==1.1.0
 proto-plus==1.15.0
 protobuf==3.12.0
 pyarrow==3.0.0
-python-dateutil==2.7.2
+python-dateutil==2.7.3
 requests==2.18.0
-Shapely==1.6.0
+Shapely==1.6.4.post2
 six==1.13.0
 tqdm==4.7.4

